pages/date-range-picker/range-calendar.page.tsx

```diff
@@ -23,7 +25,7 @@ export default function RangeCalendarScenario() {
 
         <RangeCalendar
           value={value}
-          onChange={setValue}
+          setValue={setValue}
```

Comment:
onChange has become setValue to support full React's setState signature, including update in callback